issue_processing/issue_processing.py

```diff
@@ -96,28 +100,149 @@ def reformat(issue_data):
 
     log.devinfo("Re-arranging Github issues...")
 
+    def format_time(time):
+        """Tries to format times from different sources to a consistent time format
+
+        :param time: the time that shall be formatted
+        :return: the formatted time
+        """
+
+        try:
+            # format from new GitHub-Wrapper files
+            d = datetime.strptime(time, '%Y-%m-%dT%H:%M:%SZ')
+            return d.strftime('%Y-%m-%d %H:%M:%S')
+        except ValueError:
+            # format from new GitHub-Wrapper files with no seconds
+            try:
+                d = datetime.strptime(time, '%Y-%m-%dT%H:%MZ')
+                return d.strftime('%Y-%m-%d %H:%M:%S')
+            except ValueError:
+                try:
+                    # format of relatedCommits time
+                    d = datetime.strptime(time, '%Y-%m-%dT%H:%M:%S+%f:00')
+                    return d.strftime('%Y-%m-%d %H:%M:%S')
+                except ValueError:
+                    # if the given time format is not know it can't be formatted, so it just gets returned
+                    # also happens if the time already has the correct format (old GitHub-Wrapper files)
+                    return time
+
+    # known types from JIRA and GitHub default labels
+    known_types = {"bug", "improvement", "enhancement", "new feature", "task", "test", "wish"}
+
+    # known resolutions from JIRA and GitHub default labels
+    known_resolutions = {"unresolved", "fixed", "wontfix", "duplicate", "invalid", "incomplete",
```

Comment:
`known_resolutions` and also `known_types` are predestined to be global variables. Please move them outside of this function.